fiftyone/server/metadata.py

```diff
@@ -53,10 +53,21 @@ async def get_metadata(
         metadata dict
     """
     filepath = sample["filepath"]
-    is_video = media_type == fom.VIDEO
     metadata = sample.get("metadata", None)
     urls = _create_media_urls(collection, sample, url_cache)
 
+    has_orthographic_projection_metadata = (
+        media_type == fom.POINT_CLOUD
+        and "orthographic_projection_metadata" in sample
```

Comment:
I think this should be implemented by checking for a field of type `OrthographicProjectionMetadata` rather than specifically checking for the field name that the util in this PR happens to use.

There is special logic to prevent users from renaming default fields like `filepath` and `metadata`, but there is no logic protecting the `orthographic_projection_metadata` field name right now (and I don't think we need to add that), so we should be prepared that a user might rename the field to something shorter.

Reply:
FYI I directly exposed this field as a `metadata_field` parameter of the util to make it easier for users to pick a different name if they want.

If you have concerns about type-based rather than name-based logic here, LMK